analysis/src/main/scala/io/ssc/trackthetrackers/analysis/plots/PlotTrackerDistribution.scala

```diff
@@ -47,6 +47,6 @@ object PlotTrackerDistribution extends App {
     dataset.addValue(probability, "", tracker)
   }
 
-  new SingleSeriesBarChart("tracking domain distribution", "tracking domain", "probability", Color.RED, dataset)
+  new SingleSeriesBarChart("tracking domain distribution", "tracking domain", "probability", Color.RED, dataset, null)
```

Comment:
can we make null the default value of the last parameter? than we can avoid these ugly calls